ee/tabby-ui/components/chat/chat.tsx

```diff
@@ -303,8 +303,10 @@ function ChatRenderer(
     const userMessage = qaPairs[qaPairs.length - 1].user
     // FIXME(wwayne): The first context in relevantContext is currently sent as the code query.
     //                Review and update the logic to ensure the appropriate api attribute is used.
+    const relevantContextFromActiveSelection =
+      userMessage?.relevantContext?.find(o => o.isActiveSelection)
     const contextForCodeQuery =
-      userMessage?.selectContext || userMessage?.relevantContext?.[0]
+      userMessage?.selectContext || relevantContextFromActiveSelection
```

Comment:
The relevant context from the active selection will be passed into `code_query`.